src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,60 +1,57 @@
-<div class="modal-header">
-  <button type="button"
-          class="close float-right"
-          aria-label="Close"
-          (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
-  </button>
-</div>
-<div class="modal-body">
-  <h2>
-    <img src="assets/Ceph_Logo_Stacked_RGB_120411_fa_348x348.png"
-         class="ceph-logo"
-         alt="{{ projectConstants.organization }}">
-    {{ projectConstants.organization }}
-  </h2>
-  <h3>
-    <strong>{{ projectConstants.projectName }}</strong>
-  </h3>
-  <div class="product-versions">
-    <strong>Version</strong>
-    <br>
-    {{ versionNumber }}
-    {{ versionHash }}
+<div class="about-container">
+  <div class="modal-header">
+    <button type="button" class="close float-right" aria-label="Close" (click)="modalRef.hide()">
```

Comment:
Our styleguide says one attribute per line.
```suggestion
    <button type="button" 
            class="close float-right"
            aria-label="Close"
            (click)="modalRef.hide()">
```